tests/test_regression.py

```diff
@@ -384,3 +384,53 @@ def test_escape_string(self):
                       expected_output_file="result.json",
                       script=issue285_code,
                       expected_stdout=".*Total Entries:.*")
+
+
+wait_for_child = """
+import time
+import multiprocessing
+
+def target():
+    time.sleep(3)
+
+if __name__ == '__main__':
+    p = multiprocessing.Process(target=target)
+    p.start()
+    # The main process will join the child in multiprocessing.process._children.
+    # This is a hack to make sure the main process won't join the child process,
+    # so we can test the wait_children_finish function
+    multiprocessing.process._children = set()
+    time.sleep(1)
+"""
+
+wait_for_terminated_child = """
+import time
+import os
+import signal
+import multiprocessing
+
+def target():
+    time.sleep(3)
+    os.kill(os.getpid(), signal.SIGTERM)
+
+if __name__ == '__main__':
+    p = multiprocessing.Process(target=target)
+    p.start()
+    # The main process will join the child in multiprocessing.process._children.
+    # This is a hack to make sure the main process won't join the child process,
+    # so we can test the wait_children_finish function
```

Comment:
For both comments, let's mention the class (I think it's `VizUI.wait_children_finish`?). Because we are discussing in the context of `multiprocessing` here and it's easy for people to mistaken this function as a `multiprocessing` function.